python-threatexchange/threatexchange/content_type/meta.py

```diff
@@ -28,3 +28,18 @@ def get_all_signal_types() -> t.Set[t.Type[signal_base.SignalType]]:
 @functools.lru_cache(1)
 def get_signal_types_by_name() -> t.Dict[str, signal_base.SignalType]:
     return {s.get_name(): s for s in get_all_signal_types()}
+
+
+_content_type_map = {
```

Comment:
nit: the pattern in this files seems to be to use caching instead of import/runtime creation. 

Thoughts on this instead to match? 
```
@functools.lru_cache(1)
def get_content_map():
    return content_type.get_name(): content_type for content_type in get_all_content_types()
```

Reply:
True for exported methods. For internal things like the cached map, there are no precedents.